modules/git/utils.go

```diff
@@ -35,6 +35,10 @@ func parsePrettyFormatLog(repo *Repository, logByts []byte) (*list.List, error)
 }
 
 func RefEndName(refStr string) string {
+	if strings.HasPrefix(refStr, "refs/heads/") {
+		return strings.TrimPrefix(refStr, "refs/heads/")
```

Comment:
ok, or `refStr[len("refs/heads"):]`?
Maybe we shall have a const for "refs/heads"?